components/dashboard/src/data/configurations/configuration-queries.ts

```diff
@@ -190,13 +190,9 @@ export const useCreateConfiguration = () => {
                 throw new Error("No org currently selected");
             }
 
-            // TODO: Should we push this into the api?
-            // ensure a .git suffix
-            const normalizedCloneURL = cloneUrl.endsWith(".git") ? cloneUrl : `${cloneUrl}.git`;
-
             const response = await configurationClient.createConfiguration({
                 name,
-                cloneUrl: normalizedCloneURL,
+                cloneUrl,
```

Comment:
nit: actually it's not cloneUrl anymore (xd). But cloneUrl is included in contextUrl, so it doesn't matter

Reply:
Yikes, you're right, I can fix it. It should be fine to update the API proto because the API is not used anywhere else. WDYT?